doc/source/whatsnew/v0.24.2.rst

```diff
@@ -25,6 +25,7 @@ Fixed Regressions
 - Fixed regression in :meth:`DataFrame.apply` causing ``RecursionError`` when ``dict``-like classes were passed as argument. (:issue:`25196`)
 
 - Fixed regression in :meth:`DataFrame.duplicated()`, where empty dataframe was not returning a boolean dtyped Series. (:issue:`25184`)
+- Fixed regression in :meth:`Categorical.min` and :meth:`Categorical.max` where ``numeric_only=True`` was ignored (:issue:`25299`)
```

Comment:
To nitpick, it was actually in Series with categorical data, and not in Categorical.min/max itself, I think?